content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -312,6 +322,56 @@ The value of the label is the namespace name.
 While NetworkPolicy cannot target a namespace by its name with some object field, you can use the
 standardized label to target a specific namespace.
 
+## Pod lifecycle
+
+When the NetworkPolicy rules should be applied in the pod's lifecycle is undefined, since
+there is a potential race if the pod and the NetworkPolicy that selects that pod are created
+at around the same time. Nevertheless, if the policy was created a sufficiently long time
+before the pod it selects, it is expected that all isolation rules should be set
+before any of pod containers are started. Thus, we ensure that a pod can never establish 
+an outbound connection or receive an inbound connection that is not allowed by NetworkPolicy 
+at any point in its lifecycle. At the same time allow rules may be installed later and pod 
+should be able to handle some delay before its network is fully setup.
+
+Network plugins may implement extra `status.conditions` for pod readiness based on the
+status of network policies. These conditions then may be used by 
+[Pod readiness gate](/docs/concepts/workloads/pods/pod-lifecycle/#pod-readiness-gate).
+
+## `hostNetwork` pods
+
+NetworkPolicy behaviour for `hostNetwork` pods is undefined, when
+1. `hostNetwork` pod is selected by `spec.podSelector`.
+For example, if there is a `hostNetwork` pod on the `nodeA`, that has label `role: client`,
+the following NetworkPolicy:
+```yaml
+  ...
+  spec:
+    podSelector:
+      matchLabels:
+        role: client
+  ...
+```
+may apply ingress and egress rules to all connections with `nodeA` ip, or just ignore
+that pod as if it didn't match the selector, or do something else.
```

Comment:
Please indent all this text by 3 characters; it makes the ordered list more obvious.